### ISSUE TITLE
You may be interested in libextobjc

### ISSUE THREAD

**Issue by onmyway133** [forkingdog/ProtocolKit#1]
[libextobjc](https://github.com/jspahrsummers/libextobjc)

> Concrete protocols, using EXTConcreteProtocol, for providing default implementations of the methods in a protocol.

**Comment by sunnyxx** [comment]
Yes, we have fun with `libextobjc` too, lots of codes are based on `EXTConcreteProtocol`,  this repo is built for exercising and provides a better usage like `@defs` and separated implementations. Thanks.
